src/pages/Settings/components/CategorySettings.tsx

```diff
@@ -82,6 +107,18 @@ const CategorySettings: React.FunctionComponent<Props> = ({
 			dataIndex: 'description',
 			editable: true,
 		},
+		{
+			title: '',
+			dataIndex: 'operation',
+			render: (text, record: Category) => (
+				<Popconfirm
+					title={t('categories.deleteConfirmation')}
+					onConfirm={() => handleDeleteCategory(record._id)}
```

Comment:
if you have face up with Lambdash are forbidden in JSX. You could walk around by declaring handleDeleteCategory in render function
```

	{
			title: '',
			dataIndex: 'operation',
			render: (text, record: Category) => (
                                 const handleDeleteCategory = () => deleteCategory(team._id, record._id)
				<Popconfirm
					title={t('categories.deleteConfirmation')}
					onConfirm={ handleDeleteCategory}
				>
					<Icon type="delete" theme="twoTone" twoToneColor="red" />
				</Popconfirm>
			),
		},
```

Reply:
I tried this but that lambda error is still there